patientsearch/src/js/components/TimeoutModal.js

```diff
@@ -158,23 +176,29 @@ export default function TimeoutModal() {
           <span className="error">Your current session has expired.</span>
         )}
         {expiresIn && expiresIn != 0 && (
-          <span>
-            Your session will expired in approximately{" "}
-            {getExpiresInDisplay(expiresIn)}.
-          </span>
+          <React.Fragment>
+            {!disabled && <span>
+              Your session will expired in approximately
```

Comment:
I think it's out of scope for this PR, but I did notice when I put my laptop to sleep (running Firefox 91.5.0) and later woke it up, all the javascript seemed to pause on the page. The REST calls to /validate-token stopped, and I was never redirected away from the patient list.

I'll try reproducing again and let you know if I find more.

Reply:
@ivan-c I was able to reproduce this.  I see this in the browser console, is this what you see?
<img width="1270" alt="Screen Shot 2022-01-25 at 12 02 11 PM" src="https://user-images.githubusercontent.com/12942714/151050736-a98382b2-a7e0-40eb-b340-3ff51f7cd56b.png">

Reply:
> @ivan-c I was able to reproduce this. I see this in the browser console, is this what you see? <img alt="Screen Shot 2022-01-25 at 12 02 11 PM" width="1270" src="https://user-images.githubusercontent.com/12942714/151050736-a98382b2-a7e0-40eb-b340-3ff51f7cd56b.png">

@achen2401 thanks for trying! ...but that's not what I'm seeing when I sleep/unsleep my laptop with Firefox. When it happens on my laptop, I don't see **any** console output (or REST requests)/

Reply:
@ivan-c I **think** I addressed the issue.  Will you give it a try again when you have a moment?

Reply:
Just to close the loop- tested and works great out of sleep